front/client/components/CategoryItemTraits.vue

```diff
@@ -28,7 +28,7 @@
           <category-item-btn
             slot="activator"
             size="22px"
-            :iconSize="14"
+            :iconSize="`14`"
```

Comment:
You dont need to wrap 14 in \` \` if you want it to be passed as string. Just remove `:` and it will be passed to component as string. Like this `iconSize="14"`

Reply:
done